spec/compiler/crystal/tools/doc/doc_renderer_spec.cr

```diff
@@ -374,7 +374,46 @@ describe Doc::MarkdDocRenderer do
     HTML
   end
 
-  describe "renders html" do
-    it_renders nil, %(<h1 align="center">Foo</h1>), %(<h1 align="center">Foo</h1>)
+  describe "renders html with sanitization" do
+    it_renders nil, %(<h1 align="center">Foo</h1>), %(<h1>Foo</h1>)
+    it_renders nil, %(<script>alert("hello world")</script>), %()
+    it_renders nil, %(<p style="font-size: 100px">example text</p></div>), %(<p>example text</p>)
+
+    it_renders nil, %(```crystal\n# <script>alert("hello world")</script>\n```),
+      %(<pre><code class="language-crystal"><span class="c"># &lt;script&gt;alert(&quot;hello world&quot;)&lt;/script&gt;</span></code></pre>)
+  end
+
+  describe "renders tables" do
+    it_renders nil, <<-INPUT, <<-HTML
```

Comment:
Aren't the two the same string?

Reply:
Yes they are, this is a check to ensure the sanitizer doesn't remove tables